.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -38,3 +38,4 @@ even continue reviewing your changes.
 - [ ] Chart Version bumped
 - [ ] Variables are documented in the README.md
 - [ ] Title of the PR starts with chart name (e.g. `[mychartname]`)
+- [ ] The pixie-operator-chart version is the same as the versions of olm_crd.yaml and px.dev_viziers.yaml in [Kubernetes installation recipe](https://github.com/newrelic/open-install-library/blob/main/recipes/newrelic/infrastructure/kubernetes.yml)
```

Comment:
I don't think we should add this here since this checklist is for any PR to any bundle in this repo and this is relevant only to the `nri-bundle`.